.github/workflows/drivers.yml

```diff
@@ -0,0 +1,260 @@
+name: Driver Tests
+
+on:
+  push:
+    branches:
+      - 'master'
+      - 'release-**'
+  pull_request:
+    paths-ignore:
+      - "docs/**"
+      - "**.md"
+      - "frontend/test/**"
+      - "enterprise/frontend/test/**"
+
+jobs:
+
+  be-tests-druid-ee:
+    runs-on: buildjet-2vcpu-ubuntu-2004
```

Comment:
Buildjet is chosen here instead of the (free) GitHub-hosted runner since the tests can run up to 35% faster when using the former.
But if costs become an issue, then we can go back to GitHub runner, at the expense of slightly slower test runs.

Reply:
What was the longest running time with the free GH runner?
If it's faster than the slowest E2E test, than it doesn't make sense to use BuildJet.

Reply:
We still need Buildjet for Druid and Spark tests (those need more resources than what GitHub runner can offer), but the rests are fine with GitHub runners.

Reply:
I have a feeling that Presto might need it too.